pyproject.toml

```diff
@@ -21,9 +21,10 @@ python = "^3.6"
 nautobot = "*"
 nautobot-capacity-metrics = "*"
 texttable = "^1.6.2"
-pyjwt = {extras = ["crypto"], version = "^1.7.1"}
+PyJWT = "^2.1.0"
 webexteamssdk = "^1.3"
 slack-sdk = "^3.4.2"
+cryptography = "^3.4.7"
```

Comment:
this library is used by a lot of other project, might be better to not pin its version to specific version but rather keep it flexible (if possible)

Reply:
I should have pushed harder on this one, I'm blocked because of this issue right now :(